red/Animal/scripts/barnyard.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
+
+use v5.14;
+use strict;
+use warnings;
+
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+
+our @ANIMALS = qw( Cow Horse Sheep Mouse );
```

Comment:
You'd probably only need to use ``our`` if you planned to require this into another script.